src/Uno.UI.RuntimeTests/Tests/Windows_System/Given_Launcher.cs

```diff
@@ -1,49 +1,94 @@
 ﻿#if __ANDROID__ || __IOS__
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Windows.ApplicationModel.Core;
 using Windows.System;
 using Windows.UI.Core;
-using Windows.UI.StartScreen;
 
 namespace Uno.UI.RuntimeTests.Tests.Windows_System
 {
 	[TestClass]
 	public class Given_Launcher
 	{
-		private async Task Dispatch(DispatchedHandler p)
+		[TestMethod]
+		public async Task When_Null_Uri_Is_Launched()
 		{
-			await CoreApplication.GetCurrentView().Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, p);
+			await Assert.ThrowsExceptionAsync<AccessViolationException>(() => Launcher.LaunchUriAsync(null).AsTask());
```

Comment:
it's most probably better to not throw this exception, even if it's what WinRT does. Returning an argument null exception makes more sense :)

Reply:
I'm thinking only if this would not mean that existing code which, for some reason, handles `AccessViolationException` here, as the developer didn't know the reason for it happening; would behave differently in Uno than in UWP? But agree that `ArgumentNullException` would make much more sense, and if you see it as a better solution, I will replace it there 😀

Reply:
Fixed :-)